### ISSUE TITLE
Back side of US DL 

### ISSUE THREAD

**Issue by AlibekJ** [BlinkID/blinkid-cordova#131]
I am still struggling with capturing the backside of the USDL.
If I use blinkIdCombinedRecognizer, I get front and back images, but not the USDL-specific data fields.
```javascript
let recognizerCollection = 
new cordova.plugins.BlinkID.RecognizerCollection([mrtdSuccessFrameGrabber, 
blinkIdCombinedRecognizer]);
```

If I use usdlCombinedRecognizer like this:
```javascript
let recognizerCollection = 
new cordova.plugins.BlinkID.RecognizerCollection([mrtdSuccessFrameGrabber, 
usdlCombinedRecognizer]);
```

I get all the data, but not the back image.

I tried using them both like this
```javascript
let recognizerCollection = 
new cordova.plugins.BlinkID.RecognizerCollection([mrtdSuccessFrameGrabber, 
usdlCombinedRecognizer, 
blinkIdCombinedRecognizer]);
```

However the ```blinkIdCombinedRecognizer``` picks up the data first and ```usdlCombinedRecognizer.result``` comes back empty, so, again, I get the images but not the data.

**Comment by mparadina** [comment]
Hello @AlibekJ 

Can you please tell me which specific data fields are not being returned when using the BlinkID combined recognizer?

Regards,
Milan

**Comment by AlibekJ** [comment]
@mparadina 

Here are the critical ones

let fields = usdlCombinedRecognizer.result.fields;
let USDLKeys = cordova.plugins.BlinkID.UsdlKeys;
middleName = fields[USDLKeys.CustomerMiddleName];
veteran = fields[USDLKeys.Veteran];
ssn = fields[USDLKeys.SocialSecurityNumber];
issuingState = fields[USDLKeys.IssuingJurisdictionName];

**Comment by mparadina** [comment]
Hi @AlibekJ 

The data fields you've stressed out above are specific fields that are only obtainable by the USDL recognizer. 

We do not support retrieving the specific fields with the BlinkID recognizer that can only be found on only one set of documents (for example eye color on the USDL), since the BlinkID recognizer is a generic recognizer, which contains a set of fields that are common for generally all types of documents that it currently supports reading.

However, it can return both front and back document images.

On the other hand, with the USDL recognizer, you can obtain the previously mentioned information, but it can not retrieve the backside image, since it only reads the barcode and extracts the information from it. 

Hope this was helpful.

Regards,
Milan

**Comment by AlibekJ** [comment]
That's right: I can have the back image or the data, but can not have both.
Now when you understand the problem, would appreciate if you could suggest any way of solving this.

**Comment by mparadina** [comment]
Hi @AlibekJ 

If you've decided to use the USDL recognizer, I could also suggest using the `MBCaptureHighResImage`, when it comes to taking the image of the backside of the USDL.

`MBCaptureHighResImage` can return high-resolution MBImages with capture or still output.

Documentation: http://blinkid.github.io/blinkid-ios/Type%20Definitions.html#/c:MBRecognizerRunnerViewController.h@T@MBCaptureHighResImage

Hope this helps.

Regards,
Milan
